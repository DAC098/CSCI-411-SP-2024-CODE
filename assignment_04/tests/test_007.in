3
1 1 3
gatagctggctcgggc acagaaaacaattaagtatgctttctagac
ttattagttaggtagcacggtcgtccccca aaaacgttgtcatcacccacgc
catagttattagctatggggtacccatcgc tacagttagaagtggtggccacctccac
