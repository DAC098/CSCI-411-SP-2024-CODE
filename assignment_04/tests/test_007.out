___g______a_t_ag__ctggc__tc__g_ggc
acagaaaacaattaagta_t_gctttctaga__c
22
__ttatta_gttaggtagcacggtc_gtccc_c_ca
aa__a__acgtt__gt__ca___tca__cccacgc_
20
__ca__tagttattagctatgg_gg__ta_c_ccatcgc
tacagttag__a__ag___tggtggcc_acctcca___c
20
